src/en/readallcomicscom/src/eu/kanade/tachiyomi/extension/en/readallcomicscom/ReadAllComics.kt

```diff
@@ -0,0 +1,185 @@
+        .addInterceptor(::archivedCategoryInterceptor)
+        .rateLimit(2)
+        .build()
+
+    private fun archivedCategoryInterceptor(chain: Interceptor.Chain): Response {
+        val request = chain.request()
+        val response = chain.proceed(request)
+
+        val document = Jsoup.parse(
+            response.peekBody(Long.MAX_VALUE).string(),
+            request.url.toString(),
+        )
+
+        val newUrl = document.selectFirst(".description-archive > p > span > a")
+            ?.attr("href")?.toHttpUrlOrNull()
+            ?: return response
+
+        if (newUrl.pathSegments.contains("category")) {
+            response.close()
+
+            return chain.proceed(
+                request.newBuilder()
+                    .url(newUrl)
+                    .build(),
+            )
+        }
+
+        return response
+    }
+
+    override fun popularMangaRequest(page: Int): Request {
+        throw Exception("ReadAllComics has no popular titles Page. Please use the search function instead.")
+    }
+
+    // Never called
+    override fun popularMangaFromElement(element: Element): SManga {
+        throw Exception("")
+    }
+
+    override fun popularMangaSelector() = ""
+    override fun popularMangaNextPageSelector() = ""
+
+    override fun fetchSearchManga(page: Int, query: String, filters: FilterList): Observable<MangasPage> {
+        return if (page == 1) {
+            client.newCall(searchMangaRequest(page, query, filters))
+                .asObservableSuccess()
+                .map { searchMangaParse(it) }
+        } else {
+            Observable.just(searchPageParse(page))
+        }
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList): Request {
+        val url = baseUrl.toHttpUrl().newBuilder().apply {
+            addQueryParameter("story", query)
+            addQueryParameter("s", "")
+            addQueryParameter("type", "comic")
+        }.build()
+
+        return GET(url, headers)
+    }
+
+    override fun searchMangaParse(response: Response): MangasPage {
+        searchPageElements = response.asJsoup().select(searchMangaSelector())
+
+        return searchPageParse(1)
+    }
+
+    private fun searchPageParse(page: Int): MangasPage {
+        val mangas = mutableListOf<SManga>()
+        val endRange = ((page * 24) - 1).let { if (it <= searchPageElements.lastIndex) it else searchPageElements.lastIndex }
+
+        for (i in (((page - 1) * 24)..endRange)) {
+            mangas.add(
+                searchMangaFromElement(searchPageElements[i]),
+            )
+        }
+
+        return MangasPage(mangas, endRange < searchPageElements.lastIndex)
+    }
+
+    override fun searchMangaFromElement(element: Element) = SManga.create().apply {
+        setUrlWithoutDomain(element.attr("href"))
+        title = element.text()
+        thumbnail_url = "https://fakeimg.pl/200x300/?text=No%20Cover%0AOn%20Search&font_size=62"
```

Comment:
remove this, just do empty string ("")

Reply:
keep empty string, otherwise some apps (j2k) will try to load all details in one go